sdk/core/azure-core/tests/async_tests/test_rest_stream_responses_async.py

```diff
@@ -161,10 +161,10 @@ async def test_iter_read_back_and_forth(client):
     # the reason why the code flow is like this, is because the 'iter_x' functions don't
     # actually read the contents into the response, the output them. Once they're yielded,
     # the stream is closed, so you have to catch the output when you iterate through it
-    request = HttpRequest("GET", "/basic/lines")
+    request = HttpRequest("GET", "/basic/string")
 
     async with client.send_request(request, stream=True) as response:
-        async for line in response.iter_lines():
+        async for line in response.iter_bytes():
```

Comment:
Do you still want to use "line"? It is up to you.  :)

Reply:
I'm getting rid of iter_lines so can't use it anymore, switching to iter_bytes instead

Reply:
What I meant was 

async for _**line**_ in response.iter_bytes():

:)

Reply:
ohh i see, yeah will get rid